### PR TITLE
Docs Visual Studio Packing Updates

### PR DESCRIPTION
**Pull request by kenbailey**
Add some additional tips on the Visual Studio build docs and a source reference.

### CODE REVIEW

**Comment by kenbailey** [comment]
@paulcbetts, I'm done with doc updates for a bit again. When you get a chance can you review and merge PRs #818, #819, #820, #821, #822.

Thanks!

File: docs/using/visual-studio-packaging.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ PM>  Install-Package NuGet.CommandLine
   ~~~
 * It suffers from a bug when sometimes NuGet packages are not loaded properly and throws nuget/squirrel is not recogized (9009) errors.  
  **Tip:** In this case you may simply need to restart Visual Studio so the Package Manager Console will have loaded all the package tools
+* If you get the following error you may need add the full path to squirrel.exe in the build target `Exec Command` call. `'squirrel' is not recognized as an internal or external command`
+
```
Comment (paulcbetts) [inline docs/using/visual-studio-packaging.md]: Don't think this does anything

**Comment by kenbailey** [comment]
@paulcbetts, not sure what's going on with the Github diff viewer, view the changes inline and you'll see the actual edits to the docs include the following:

> 
>  If you get the following error you may need add the full path to squirrel.exe in the build target Exec Command call. 'squirrel' is not recognized as an internal or external command
> 
> Source: Issue #630

 Really strange. I can't see anything wrong with the files.

**Comment by paulcbetts** [comment]
xddddddddddddddddddddddddddddddddzzsszdcx------

**Pull request merged**